server/controllers/list.js

```diff
@@ -875,19 +888,23 @@ const updateItemDetails = (req, resp) => {
         itemToUpdate.link = link;
       }
 
-      if (isOrdered !== null) {
+      if (isOrdered !== undefined) {
```

Comment:
check if this will be enough `if(!isOrdered) `

Reply:
It won't be enough, because sometimes this prop isn't passed to this controller, in solution: `if(!isOrdered)` if isOrdered or isArchived prop would be false item wouldn't be updated...